src/app/Authenticated.tsx

```diff
@@ -98,15 +103,21 @@ const Authenticated = () => {
     if (isValidating) {
         return <FullPageSpinner />;
     } else if (combinedGrants.length === 0) {
-        return <NoGrantsFound />;
-    } else {
         return (
-            <AuthenticatedOnlyContext>
-                <Routes>
+            <Routes>
+                <Route path={routeDetails.user.path}>
                     <Route
-                        path={routeDetails.registration.path}
+                        path={routeDetails.user.registration.path}
                         element={<Registration />}
                     />
+                </Route>
+                <Route path="*" element={<NoGrantsFound />} />
+            </Routes>
```

Comment:
Making folks that hit "No Grants" able to land on the registration page as that will be who we need to "register"